lib/AppInfo/Application.php

```diff
@@ -145,10 +145,28 @@ function () {
 		}
 
 		if ($this->publicLinksAllowedToUseCollabora()) {
-			\OCP\Util::connectHook(Share::class, "share_link_access", \OCA\Richdocuments\HookHandler::class, "addViewerScripts");
+			Util::connectHook(Share::class, "share_link_access", $this, "addViewerScripts");
 		}
 
-		\OCP\Util::connectHook('\OCP\Config', 'js', \OCA\Richdocuments\HookHandler::class, 'addConfigScripts');
+		Util::connectHook('\OCP\Config', 'js', $this, 'addConfigScripts');
+	}
+
+	public function addViewerScripts() {
+		Util::addScript('richdocuments', 'viewer/viewer');
+		Util::addStyle('richdocuments', 'viewer/odfviewer');
+	}
+
+	public function addConfigScripts($array) {
+		$appConfig = $this->getContainer()->query(AppConfig::class);
+		$array['array']['oc_appconfig']['richdocuments'] = [
```

Comment:
Include some comment around because it isn't clear that this will modify the array passed as parameter.

Reply:
added as PHPDoc